.github/workflows/ci.yml

```diff
@@ -0,0 +1,114 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    strategy:
+      matrix:
+        go-version: [1.15.x, 1.14.x]
+        platform: [ubuntu-latest, macos-latest]
+        #platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Do GitHub actions not work on Windows? Shall we just delete this?

Reply:
I guess there are some `clrf` trickery to apply for Windows tests to pass.

`appveyor` is running great for now (at least for gonum/plot).
let's live with that and leave it at that.

Reply:
Yes, I recall seeing that GHA have `core.autocrlf=true`, something that I fixed in travis's build system (it's configurable there). You can have local files specifying the behaviour apparently (this is how it should work, but travis ignored those).

Here's the issue: https://github.com/actions/checkout/issues/135